models/marts/ledger_current_state/config_settings_current.yml

```diff
@@ -3,7 +3,15 @@ version: 2
 models:
   - name: config_settings_current
     description: '{{ doc("config_settings_current") }}'
-
+    tests:
+      - dbt_utils.recency:
+          datepart: hour
+          field: closed_at
+          interval: 12
+          config:
+            severity: '{{ "error" if target.name == "prod" else "warn" }}'
```

Comment:
config setting updates are rare occurrences. I think we've changed the settings [4 times](https://stellar.expert/explorer/public/protocol-history) since June. We shouldn't have a freshness check on this table as the settings updates are ad hoc

Reply:
Makes sense. In that case, I can make this an exception in our test coverage or can also make threshold really high

Reply:
i would just make an exception. We can't reliably predict when updates to config settings occur